### ISSUE TITLE
Enhancement: Easier way of duplicating objects in the Outline view. Right click > Duplicate

### ISSUE THREAD

**Issue by alexandru-p** [defold/editor2-issues#226]
Sometimes, when multiple similar items are required (for example: multiple similar collision object shapes, sprites, etc.), it would be nice to be able to right click an already created one in the Outline View and duplicate it, along side with it's properties.

**Comment by ragnard-king** [comment]
Requested again, but including asset browser, on forum:
https://forum.defold.com/t/ctrl-d-to-duplicate-assets-objects-nodes/87

**Comment by nemish** [comment]
Or may be add child objects by drag'n'drop from assets to outline?

**Comment by ragnard-king** [comment]
@nemish Yep, another good suggestion, previously logged here: https://github.com/defold/editor2-issues/issues/408
